src/pip/_internal/utils/glibc.py

```diff
@@ -14,6 +14,19 @@ def glibc_version_string():
     # type: () -> Optional[str]
     "Returns glibc version string, or None if not using glibc."
 
+    try:  # https://github.com/pypa/pip/issues/6675#issue-463147612
```

Comment:
Instead of including a link, I would include a stand-alone comment explaining what the logic is for. That way people don't need to click a link to get an understanding. Also, if this code is being copy-and-pasted from somewhere (the standard library), I would include a link to that code in the CPython repo. The link to the issue will be findable via the PR this commit comes from.